x-pack/platform/test/api_integration/apis/aiops/types.ts

```diff
@@ -12,7 +12,7 @@ import type {
 import type { SignificantItem, SignificantItemGroup } from '@kbn/ml-agg-utils';
 import type { FetchFieldCandidatesResponse } from '@kbn/aiops-log-rate-analysis/queries/fetch_field_candidates';
 
-import type { LogRateAnalysisDataGenerator } from '../../../functional/services/aiops/log_rate_analysis_data_generator';
+import type { LogRateAnalysisDataGenerator } from '../../services/aiops/log_rate_analysis_data_generator';
```

Comment:
I copied some services from "functional tests" in the new structure so that we don't keep common ones in the single place: `x-pack/platform/test/api_integration/services`

When we relocate functional tests, we will extend this common services with browser/UI testing services. This way we solve cross imports api -> functional and functional -> api